ts/features/pn/aar/components/SendAARCieCardReadingComponent.tsx

```diff
@@ -0,0 +1,140 @@
+import { useCallback, useEffect, useMemo } from "react";
+import i18n from "i18next";
+import {
+  ReadStatus,
+  useCieInternalAuthAndMrtdReading
+} from "../hooks/useCieInternalAuthAndMrtdReading";
+import {
+  CieCardReadContent,
+  CieCardReadContentProps
+} from "../../../common/components/cie/CieCardReadContent";
+import { useIODispatch } from "../../../../store/hooks";
+import { RecipientInfo, sendAARFlowStates } from "../utils/stateUtils";
+import { setAarFlowState } from "../store/actions";
+import { isDefined } from "../../../../utils/guards";
+import { useSendAarFlowManager } from "../hooks/useSendAarFlowManager";
+
+type ScreenContentProps = Omit<CieCardReadContentProps, "progress">;
+
+export type SendAARCieCardReadingComponentProps = {
+  iun: string;
+  mandateId: string;
+  recipientInfo: RecipientInfo;
+  can: string;
+  verificationCode: string;
+};
+
+export const SendAARCieCardReadingComponent = ({
+  can,
+  verificationCode,
+  iun,
+  recipientInfo,
+  mandateId
+}: SendAARCieCardReadingComponentProps) => {
+  const dispatch = useIODispatch();
+  const { terminateFlow } = useSendAarFlowManager();
+  const { startReading, stopReading, readStatus, progress, data, error } =
+    useCieInternalAuthAndMrtdReading();
+
+  const handleStartReading = useCallback(() => {
```

Comment:
Do we need to create a wrapper around this function? What are the cons of calling `startReading(...)` directly?